galasa-extensions-parent/dev.galasa.ecosystem.builder/src/main/java/galasa/ecosystem/docker/builder/DockerEcosystemBuilder.java

```diff
@@ -0,0 +1,628 @@
+package galasa.ecosystem.docker.builder;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.Scanner;
+
+public class DockerEcosystemBuilder {
+
+	public static void main(String[] args) throws FileNotFoundException, UnsupportedEncodingException {
+		
+		  String subnet = null;
+		  String gateway = null;
+		  String apiName = null;
+		  String netName = null;
+		  String apiResPol = null;
+		  String apiVol = null;
+		  String apiVolSrc = null;
+		  String apiVolTrg = null;
+		  String apiPort = null;
+		  String contName = null;
+		  String contResPol = null;
+		  String contVol = null;
+		  String cpsName = null;
+		  String cpsResPol = null;
+		  String cpsVolSrc = null;
+		  String cpsVolTrg = null;
+		  String cpsPort = null;
+		  String cdbName = null;
+		  String cdbResPol = null;
+		  String cdbEnv = null;
+		  String cdbVolSrc = null;
+		  String cdbVolTrg = null;
+		  String cdbPort = null;
+		  String resmonName = null;
+		  String resmonResPol = null;
+		  String resName = null;
+		  String resResPol = null;
+		  String resPort = null;
+		  String sbName = null;
+		  String sbResPol = null;
+		  String sbPort = null;
+		
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Welcome to the Galasa Ecosystem for Docker set up" + "\n");
+		System.out.println("Would you like to reuse old config (default Y):");
+		String reuse = scanner.nextLine();
+	    while (!(reuse.toUpperCase().equals("Y") || reuse.toUpperCase().equals("N") || reuse.isEmpty())) {	
+			System.out.println("Please enter Y or N");
+			reuse = scanner.nextLine();
+		}
+		if ((reuse.toUpperCase().equals("Y") || (reuse.isEmpty())))	{
+			System.out.println("Using previously set config");	
```

Comment:
This should present a list of options, their current values and give them the opportunity to change individual options.
At the moment,  if they make a mistake in a reply,  they have to ctrl+c out and start all over,  not very user friendly.